cart.py

```diff
@@ -0,0 +1,40 @@
+from global_vars import *
+from collections.abc import Iterable
+
+
+class Cart(Iterable):
+    def __init__(self):
+        self.cart = {}
+
+    # It allows to iterate on sku's and quantities
+    def __iter__(self):
+        return iter(self.cart.items())
+
+    def add(self, sku: str, qty: int = 1):
+        if sku not in food_menu:
+            raise ValueError("Invalid SKU. Please check that the entered menu item is correct")
```

Comment:
I would remove the second part of the message. Also, the Sku needs to be printed in the message (same for other exception messages below
)